config/ember-try.js

```diff
@@ -1,11 +1,36 @@
 /*jshint node:true*/
 module.exports = {
-  useVersionCompatibility: true,
```

Comment:
I do agree that staying in sync with the community is a good idea here, will remove.